pokemongo_bot/cell_workers/move_to_fort.py

```diff
@@ -71,17 +74,21 @@ def work(self):
 
             if self.is_attracted() > 0:
                 fort_event_data.update(lure_distance=format_dist(self.lure_distance, unit))
-                self.emit_event(
-                    'moving_to_lured_fort',
-                    formatted="Moving towards pokestop {fort_name} - {distance} (attraction of lure {lure_distance})",
-                    data=fort_event_data
-                )
+                if int(self.prev_dist/self.config.get("log_interval_meters", 10)) != int(dist/self.config.get("log_interval_meters", 10)):
```

Comment:
If we're going to go ahead with this PR, can we fetch "log_interval_meters" once? We use it twice in this `if` alone.